server/sonar-server/src/main/java/org/sonar/server/webhook/WebhookQGChangeEventListener.java

```diff
@@ -22,6 +22,8 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
+import org.apache.commons.lang.StringUtils;
+import org.sonar.core.util.stream.MoreCollectors;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.core.util.stream.MoreCollectors'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FSONAR-10117%2Flive-measures&issues=AWDW84WGJC-F6PRGSYoD&open=AWDW84WGJC-F6PRGSYoD)

[ISSUE_KEY]: AWDW84WGJC-F6PRGSYoD